opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -688,7 +694,7 @@ class WellModelMatrixAdapter : public Dune::AssembledLinearOperator<M,X,Y>
                     pattern[idx].insert(*wc);
 
                 // Add just a single element to ghost rows
-                if (elem.partitionType() != Dune::InteriorEntity)
+                if (elem.partitionType() == Dune::GhostEntity)
```

Comment:
This is totally unrelated to this PR but it just struck me when reading the diff:

Should we move the for-loop over the wellConnectionsGraph_ to else branch and explicitly use `noGhostMat_->setrowsize(1);`. That seems clearer and might prevent surprises if we distribute the wells.